src/components/ConvictionResults.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Button, List, Table } from 'semantic-ui-react';
+
+const Headers = [
```

Comment:
I believe the lawyer wanted to carry the "is domestic violence" flag to the result page since it has impact on how judge sees the case.

Reply:
'is domestic violence' checkbox should be in both places (Conviction and ConvictionResults)?

Reply:
Yes, but as mentioned on slack I think you should hold off on any modifications until the revised wireframe comes in from ayesha (slack handle)

Reply:
Yes, no hurry.